test/util/e2e.go

```diff
@@ -400,3 +400,11 @@ func WaitForPodRunning(ctx context.Context, k8sClient client.Client, pod *corev1
 		g.Expect(createdPod.Status.Phase).To(gomega.Equal(corev1.PodRunning))
 	}, LongTimeout, Interval).Should(gomega.Succeed())
 }
+
+func UpdateKueueConfiguration(ctx context.Context, k8sClient client.Client, config *configapi.Configuration, kindClusterName string, applyChanges func(cfg *configapi.Configuration)) {
+	configurationUpdate := time.Now()
+	applyChanges(config)
+	ApplyKueueConfiguration(ctx, k8sClient, config)
+	RestartKueueController(ctx, k8sClient, kindClusterName)
+	ginkgo.GinkgoLogr.Info("Kueue configuration updated", "took", time.Since(configurationUpdate))
+}
```

Comment:
We already have this function. Can we reuse it? 

https://github.com/kubernetes-sigs/kueue/blob/1af6b2be356e6228334d71048934e8d036798cb7/test/e2e/customconfigs/suite_test.go#L82-L89